packages/reactstrap-validation-select/src/AvSelect.js

```diff
@@ -259,6 +261,17 @@ class AvSelect extends AvBaseInput {
       Tag = Creatable;
     }
 
+     // Enhance placeholder for accessibility
+  placeholder = (
```

Comment:
Since `placeholder` is defined as `const` we can't reassign it here. We will either need to create a new variable or destructure `placeholder` separately